sensor/tests/resource/service/service_test.go

```diff
@@ -200,27 +207,48 @@ func (s *DeploymentExposureSuite) Test_NodePortPermutation() {
 				),
 				"'PortConfig' for Node Port service test not found",
 			)
-			testC.LastViolationState(nginxDeploymentName,
+			testC.LastViolationState(deploymentName,
 				assertAlertTriggered(
 					&storage.Alert{
 						Policy: &storage.Policy{
-							Name: servicePolicyName,
+							Name: policyName,
 						},
 						State: storage.ViolationState_ACTIVE,
 					},
 				),
-				fmt.Sprintf("Alert '%s' should be triggered", servicePolicyName))
+				fmt.Sprintf("Alert '%s' should be triggered", policyName))
 			testC.GetFakeCentral().ClearReceivedBuffer()
-		},
-	)
+		})
 }
 
-func (s *DeploymentExposureSuite) Test_LoadBalancerPermutation() {
-	s.testContext.RunWithResourcesPermutation(
-		[]resource.YamlTestFile{
-			NginxDeployment,
-			NginxServiceLoadBalancer,
-		}, "LoadBalancer", func(t *testing.T, testC *resource.TestContext, _ map[string]k8s.Object) {
+func (s *DeploymentExposureSuite) Test_NodePortPermutation() {
```

Comment:
Is there a way to write this as a table test? Maybe with a test case struct like: 
```go
type testCase struct {
    port int
    orderedResources []resource.YamlTestFile
}
```

Reply:
I think it would also make sense to write a comment on why we need to use different port numbers. Like you wrote in the PR description.

Reply:
Done.